src/quo/components/wallet/progress_bar/view.cljs

```diff
@@ -2,7 +2,16 @@
   (:require
     [quo.components.wallet.progress-bar.style :as style]
     [quo.theme :as quo.theme]
-    [react-native.core :as rn]))
+    [react-native.core :as rn]
+    [schema.core :as schema]))
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:full-width? [:maybe :boolean]]]]
+    :any]])
```

Comment:
There's more props that are passed to his component than this (check the styles ns)